dll/cpl/appwiz/lang/ro-RO.rc

```diff
@@ -94,5 +94,5 @@ BEGIN
     IDS_NO_DIRECTORY "Nu a fost dat niciun director!"
     IDS_INVALID_PATH "Calea dată este nevalidă!"
     IDS_INVALID_NAME "Numele comenzii rapide pe care l-ați introdus conținea fie caractere nevalide pentru numele fișierelor, fie era prea lung."
-    IDS_BROWSE_FOR_TARGET "Please select the target of the shortcut below:"
+    IDS_BROWSE_FOR_TARGET "Vă rugăm să selectați ținta comenzii rapide de mai jos:"
```

Comment:
This is the textbook example of excessive politeness, which should be avoided ([see rule №7 from Romanian translation notes](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt#L19)).
This principle was taken from the (Romanian) software translator's guide, that although is referenced, in the end-note list of that file, the original link is no longer valid (and should be updated). A copy of that original (rule in that) guide can be found here: [https://diacritice.sourceforge.io/ghid.html](https://diacritice.sourceforge.io/ghid.html#Polite.C8.9Bea_excesiv.C4.83)